bff-service/src/app.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { All, CacheInterceptor, Controller, Req, Request, UseInterceptors } from "@nestjs/common";
+import { AppService } from './app.service';
+
+@Controller()
+@UseInterceptors(CacheInterceptor)
```

Comment:
According to the task, only the products list request should be cached.
It is a bad idea to cache cart for 60 seconds